api/cmd/managerapi.go

```diff
@@ -119,5 +132,28 @@ func NewManagerAPICommand() *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVarP(&conf.WorkDir, "work-dir", "p", ".", "current work directory")
+
+	cmd.AddCommand(newStopCommand())
+	return cmd
+}
+
+func newStopCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "stop",
+		Run: func(cmd *cobra.Command, args []string) {
+			pid, err := utils.ReadPID(conf.PIDPath)
+			if err != nil {
+				if syscall.ENOENT.Error() == err.Error() {
+					fmt.Fprintf(os.Stderr, "failed to get manager-api pid: %s\n", err)
+				} else {
+					fmt.Fprintf(os.Stderr,  "pid path %s not found, is manager-api running?\n", conf.PIDPath)
+				}
```

Comment:
I think we should log the error info here.

`syscall.ENOENT.Error() == err.Error()` ? 
Maybe `syscall.ENOENT.Error() != err.Error()`

Reply:
Yes.

Reply:
Fixed.